config/locales/models/users/en.yml

```diff
@@ -43,5 +43,6 @@ en:
         one: Grader
         other: Graders
       user:
+        deleted: "(deleted)"
```

Comment:
I think this is more general. Please move this label to be under `common/en.yml`, and name it `deleted_placeholder`, since it isn't just the word "deleted" but the parentheses as well